src/go/rpk/pkg/cli/cmd/container/start.go

```diff
@@ -296,10 +296,18 @@ func startCluster(
 	for _, node := range nodes {
 		brokers = append(brokers, node.addr)
 	}
-	log.Infof(
-		"\nCluster started! You may use rpk to interact with it."+
-			" E.g:\n\nrpk cluster info --brokers %s\n", strings.Join(brokers, ","),
-	)
+	m := `
+Cluster started! You may use rpk to interact with it. E.g:
+
+  rpk cluster info --brokers %s
+
+Alternatively, you may set an environment variable with the comma-separated list of broker addresses:
```

Comment:
`Alternatively, you may` => `You may also`

A person can still use the brokers flag if the environment variable exists (flag overrides env).

Also, I think the spaces in the output are nice, but spaces prevent commands from showing up in shell history, so a person may not want to just c&p this output. How do other commands (gh?) do something similar?

Reply:
fwiw I'm leaning to keep the spaces, but I think we should confirm

Reply:
Based off of `git` output including spaces, we're leaning to keep the spaces